app/src/main/java/com/huji/foodtricks/foodtricks/ChooseIngredients.java

```diff
@@ -34,8 +46,11 @@
     private static final int MAX_RECIPES = 10;
     static final String INGREDIENTS = "Ingredients";
     static final String COOKING_TIME = "CookingTime";
+    static final Integer ITEM_PRESSED = 70;
+    static final Integer ITEM_UNPRESSED = 255;
 
     private IngredientsList ingredientsList;
+    private Integer[] COLOURS = new Integer[5];
```

Comment:
probably should be `COLORS` or `COLORS_ARRAY` (without the u in colors)

Reply:
"Colours" with a U is correct but British.

Reply:
I will end U